phpunit/bootstrap.php

```diff
@@ -186,6 +186,18 @@ function gutenberg_register_test_block_for_feature_selectors() {
 }
 tests_add_filter( 'init', 'gutenberg_register_test_block_for_feature_selectors' );
 
+// Used in class-wp-theme-json-test.php to mock theme file URIs.
+function gutenberg_tests_filter_theme_file_uri( $file ) {
```

Comment:
Making testing this feature in the theme-json test class easier 😄

Reply:
Oh, won't passing the relative url of an actual file in the test theme folder work?

Reply:
Good question!

It does, but the path is something really long and non-web-URLy-like. From memory it was a local file path something like `/user/something/phpunittests/...data/tests/path/to/theme`

I wasn't sure whether that path would change per test environment so to be safe I used the filter.

Reply:
Oh I see, because we're outputting an absolute url. It feels very non-obvious though, having the test setup add a filter like this. Would it be worth adding a comment in the test itself explaining how it works? That way anyone editing these tests can know what's going on behind the scenes.

Reply:
>  It feels very non-obvious though, having the test setup add a filter like this. 

That's a fair point. 

What about if I just added/removed the filter in the test itself? I started doing that and got too clever for myself 😄 

Only one test relies on this, so maybe it's only worth adding to tests if others need something similar.

Reply:
Yeah that's a great point; adding the filter in the test is a way to both make it more visible, and avoid side-effects on other tests.